xclim/core/indicator.py

```diff
@@ -289,6 +290,9 @@ class Indicator(IndicatorRegistrar):
     freq = None
     allowed_periods = None
 
+    # A dictionary of suspect flag-raising criteria
+    flag = {"no-check": lambda x: False}
```

Comment:
Woups, was this line forgotten? I believe we made the decision to take the data flags thing out of the indicator object?